parameters/example_model_params_lockdown.json

```diff
@@ -6,9 +6,18 @@
     "nIters":1,
     "rngSeed":42,
     "outputDirectory":".",
-    "lockDown":{
-        "start":55,
-        "end":150,
-        "socialDistance":0.5
-    }
+    "lockdownComponents": [
+        {
+            "type" : "FullLockdown",
+            "start": 20,
+            "end": 40,
+            "args": [2.2]
```

Comment:
Since we want these to be configurable in the model files there's not a great story for documenting these args well. For example this 2.2 is a "social distancing" constant - not sure there's a great way around this (trying to parse different shapes based on the type is tricky).

Reply:
Makes sense.  (But we definitely want some clear use cases to figure out if this is the right solution, or if something less general/flexible might be better.)